### PR TITLE
Remove themes since we have an awesome one now that fits all

### PR DESCRIPTION
**Pull request by RickvdStaaij**
### What

Remove themes since we have an awesome one now that fits all.

### Why

- Supporting multiple templates was a pain
- There is one perfect theme now

### CODE REVIEW

**Comment by RickvdStaaij** [comment]
image

File: front-end/sass/_colors.sass [hunk 1]
```diff
@@ -1,17 +1,21 @@
 $color-info: #64a9e8
 $color-info-dark: darken($color-info, 5%)
+$color-info-darker: darken($color-info, 15%)
 $color-info-light: lighten($color-info, 5%)
 
 $color-error: #e57064
 $color-error-dark: darken($color-error, 5%)
+$color-error-darker: darken($color-error, 15%)
```
Comment (RickvdStaaij) [inline front-end/sass/_colors.sass]: This one is really extreme, lets make it 10%

Comment (RickvdStaaij) [reply front-end/sass/_colors.sass]: Fixed.

File: front-end/components/Dashboard/dashboard.vue [hunk 2]
```diff
@@ -1,9 +1,7 @@
 <template>
     <div>
         <div class="dashboard" :class="theme">
```
Comment (steefmin) [inline front-end/components/Dashboard/dashboard.vue]: I think this class is not needed anymore

Comment (RickvdStaaij) [reply front-end/components/Dashboard/dashboard.vue]: Nicely spotted sir

Comment (RickvdStaaij) [reply front-end/components/Dashboard/dashboard.vue]: It dead

**Pull request merged**